### PR TITLE
docs(README, CONTRIBUTING): added cypress and its usage

### PR DESCRIPTION
**Pull request by Sboonny**
Co-authored-by: Moshe <shootermv@gmail.com>

- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

As mentioned in the discord, it will be beneficial to chapter if we mention cypress and its usage

here is how the `README.md` currently looks https://github.com/Sboonny/chapter/tree/docs/add-cypress-to-documents

here is how the `CONTRIBUTING.md` currently looks https://github.com/Sboonny/chapter/tree/docs/add-cypress-to-documents

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,58 +14,63 @@ All of an _organization_'s user data will remain under their control.
...
-| _administrator_ | the _role_ of a _user_ who can setup and manage _chapters_ and _organizers_ for an _organization_ | Women Who Code - European Administrator |
-| _organizer_ (not MVP) | the _role_ of a _user_ who can manage a _chapter's_ _events_, RSVPs, communications, and _members_ | Women Who Code - Edinburgh, Local Organizer |
-| _member_ | the _role_ of a _user_ who can follow and receive notifications from a _chapter_ and RSVP to _events_  | Women Who Code - Edinburgh, Local Member |
+| Term                  | Definition                                                                                                                                                                             | Example                                                        |
+| --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------- |
```
Comment (allella) [inline README.md:21]: @Sboonny there's a lot of extra whitespace and dashes added to the Terminology table. Was the intentional? It doesn't seem to change the way the table looks and I wonder if there's benefit to the extra characters.

Comment (Sboonny) [reply README.md:21]: It wasn't intentional, I think it's viscose auto format on my end😅. Let me revert it

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Are there any changes to the README other than formatting?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -413,6 +413,8 @@ In order to understand where to start, it may help to familiarize yourself with
...
 
 The Chapter client uses the React framework [Next.js](https://nextjs.org/) with [Apollo Client](https://www.apollographql.com/docs/react/) for data fetching.  Since we are generating a GraphQL schema we can use [GraphQL Code Generator](https://www.graphql-code-generator.com/) to convert the schema into a set of TypeScript types and, more importantly, functions to get the data from the server.  As a result we know exactly what we're allowed to request from the server and the shape of the data it returns.
+
+After you have added new feature, to not lose your sanity while testing multiple possiblites, we recommend using [Cypress](https://www.cypress.io/). It will help you testing different scenarios, created using the data from the schema, and preformed in the client side of the application, while warning you if you didn't receive certain outcome.
```
[the remark this answers is not in the document]
Comment (Sboonny) [reply CONTRIBUTING.md]: > Are there any changes to the README other than formatting?

Add link to cypress in tech used

[the remark this answers is not in the document]
Comment (allella) [reply CONTRIBUTING.md]: Only thing I see is in the last suggestion has "you if you" in the last line and the last "you" can be removed if we're going with that version.

[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply CONTRIBUTING.md]: Thanks, Jim, I typed that too quickly.

**Review by ojeytonwilliams — APPROVED** [review]

**Pull request merged**